src/controllers/awsS3/index.js

```diff
@@ -0,0 +1,28 @@
+import AWS from 'aws-sdk'
+
+export const signS3 = async (obj, { input: { filename, filetype } }) => {
+  const s3 = new AWS.S3()
+
+  s3.config.update({
+    accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+    secretAccessKey: process.env.AWS_ACCESS_KEY
+  })
+
+  const s3Bucket = process.env.AWS_BUCKET
```

Comment:
By putting this piece of code inside your resolver, you'll be creating and configuring a new S3 instance for every request. The configuration will not change unless the server is restarted, because you're using environment variables. Consider moving this part of the code to the module's top-level so that the S3 instance is created and configured only once at server startup.

Reply:
Interesting, you're right